src/templates/best-cards/generic/animalism.html

```diff
@@ -1,115 +1,97 @@
 {% extends "best-cards/_layout.html" %}
+{% import "best-cards/_layout_card_column.html" as layout with context %}
 
 {% block content %}
 <h1><i>i</i> Animalism</h1>
 <div class="tip">
+	{% trans trimmed
+        read_winds=card("Read the Winds")
+    %}
```

Comment:
Also use `{{ ani }}` and `{{ aus }}` instead of `<i>i</i>` and `<i>a</i>`. Disciplines are in the Jinja context, no need for variables in the `trans` block.